images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
+      com.redhat.component="$NAME" \
+      name="$FGC/$NAME" \
```

Comment:
I suspect that this should be `name="$NAME"`.

`FGC` is a Fedora thing. It used to be recommended by a previous version of the [Fedora container guidelines](https://docs.fedoraproject.org/en-US/containers/guidelines/creation/), apparently not anymore.

Reply:
Agreed. See upcoming update to PR.